packages/forms/frontend/sirius-components-forms/src/toolbaraction/ToolbarAction.tsx

```diff
@@ -186,7 +186,7 @@ export const ToolbarAction = ({ editingContextId, formId, widget, readOnly }: To
         onClick={onClick}
         onBlur={onBlur}
         onFocus={onFocus}
-        disabled={readOnly}
+        disabled={readOnly || !widget.readOnly}
```

Comment:
Why this one is `!widget.readOnly` while all other are `widget.readOnly`?

Reply:
a mistake, I'll change that